e2e/lease_count_test.go

```diff
@@ -0,0 +1,327 @@
+package e2e
+
+import (
+	"context"
+	"strconv"
+	"testing"
+	"time"
+
+	"github.com/google/uuid"
+	coordinationv1 "k8s.io/api/coordination/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/e2e-framework/pkg/envconf"
+	"sigs.k8s.io/e2e-framework/pkg/features"
+)
+
+type leaseController struct {
+	labels        map[string]string
+	validLeases   []*coordinationv1.Lease
+	expiredLeases []*coordinationv1.Lease
+}
+
+func NewLeaseController(labels map[string]string) *leaseController {
```

Comment:
e2e/lease_count_test.go:22:51: unexported-return: exported func NewLeaseController returns unexported type *e2e.leaseController, which can be annoying to use (revive)
func NewLeaseController(labels map[string]string) *leaseController {
                                                  ^
                                                  
Is there a reason NewLeaseController needs to be public? If not its probably easier to just make it not public.